jdaviz/configs/default/plugins/gaussian_smooth/gaussian_smooth.py

```diff
@@ -54,7 +54,8 @@ def _on_data_selected(self, event):
             return
 
         # NOTE: if this is ever used anywhere else, it should be moved into DatasetSelect
-        self.selected_data_is_1d = len(self.dataset.selected_dc_item.data.shape) == 1
+        if self.dataset.selected_dc_item is not None:
+            self.selected_data_is_1d = len(self.dataset.selected_dc_item.data.shape) == 1
```

Comment:
what scenario was causing this to trigger when it was still None?  (I'm just wondering if we need to handle defaulting `self.selected_data_is_1d` in this case, whereas the logic now will just leave it at its current value).

Reply:
ok... that's probably a case that should be handled better across all plugins, but I think this is fine as-is for now